src/bridges/zora/ZoraBridge.sol

```diff
@@ -155,26 +147,27 @@ contract ZoraBridge is BridgeBase {
             if (_inputAssetA.assetType != AztecTypes.AztecAssetType.VIRTUAL) {
                 revert ErrorLib.InvalidInputA();
             }
-            // Output type needs to be ETH.
-            if (_outputAssetA.assetType != AztecTypes.AztecAssetType.ETH) {
+            // Output type needs to be VIRTUAL.
+            if (_outputAssetA.assetType != AztecTypes.AztecAssetType.VIRTUAL) {
                 revert ErrorLib.InvalidOutputA();
             }
 
+            uint256 inputAssetId = _inputAssetA.id;
```

Comment:
https://github.com/michaelneuder/aztec-connect-bridges/pull/12/files#r1065226502

Reply:
i was getting a stack too deep error from using _inputAssetA.id every time. i guess it adds an element to the stack every time it accesses the .id field? I will look at it again to see if i can get rid of that error some other way.